src/styles/blocks/form.scss

```diff
@@ -0,0 +1,49 @@
+.form {
+  display: flex;
+  flex-direction: column;
```

Comment:
Are you sure you have to use this here? Elements are already placed in columns.

Reply:
Default behavior of flex  is row

Reply:
I meant, when you don't use flex at all, block elements are placed in column.